### PR TITLE
Header/Intro/Info page

### PR DESCRIPTION
**Pull request by mikeyrayvon**
This is all of the content and layout beyond the clapping itself.

Probably could add some comments, but here is the branch if you want to check it out. 

From here we basically just need to fix the audio and we're done

### CODE REVIEW

File: src/components/App.js [hunk 1]
```diff
@@ -1,13 +1,186 @@
...
+    this.handleInfoButtonClick = this.handleInfoButtonClick.bind(this);
+    this.handleBackgroundColor = this.handleBackgroundColor.bind(this);
+    this.handleInstructionsButtonClick = this.handleInstructionsButtonClick.bind(this);
+    this.handleFirstTouch = this.handleFirstTouch.bind(this);
+  }
+
+  handleBackgroundColor() {
+    this.setState({
+      backgroundColor: randomColor({
+        luminosity: 'bright'
```
Comment (varffs) [inline src/components/App.js]: why not bright on init too?

File: src/components/App.js [hunk 2]
```diff
@@ -1,13 +1,186 @@
...
+        backgroundColor: 'white',
+        color: this.state.backgroundColor
+      }
+    }
+
     return (
-      <div>
-        <XYController />
+      <div
+        onMouseDown={this.handleFirstTouch}
```
Comment (varffs) [inline src/components/App.js]: is there a handler like e.g. onTap that does both these in one?

**Comment by varffs** [comment]
I've added config based fixes for my issues 1 & 3. 2 was just a comment

File: src/components/App.js (hunk 1 above)
Comment (mikeyrayvon) [reply src/components/App.js]: true. fixing

File: src/components/App.js (hunk 2 above)
Comment (mikeyrayvon) [reply src/components/App.js]: not that i can find

**Pull request merged**